.husky/pre-commit

```diff
@@ -1,5 +1,2 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
-
-node ./scripts/sanityCheckEnLyaml.js
```

Comment:
A little historical context on these lang changes - with `cli-lib` there was no bundling process and the `lyaml` files didn't get shipped with the package, so I had set it up to download the lang data on commit to keep it updated and then we could bundle it like this. No longer necessary with `local-dev-lib` bundling its own lang data so we don't see those failures